assets/app/styles/_core.less

```diff
@@ -763,3 +763,11 @@ labels + dl {
   margin-top: 21px;
 }
 
+h3 + hr {
```

Comment:
I wonder if h3 + hr shouldn't just be an h3 bottom border on these pages so we always have consistent header styles and don't have to add hr to the markup. Maybe have a class for each of these pages, `.resource-details`, and a `.resource-details h3` style?